src/genie/libs/parser/iosxr/show_interface.py

```diff
@@ -1576,7 +1576,8 @@ class ShowIpv6VrfAllInterface(ShowIpv6VrfAllInterfaceSchema):
     """Parser for show ipv6 vrf all interface"""
 
     cli_command = ['show ipv6 vrf {vrf} interface {interface}',
-                   'show ipv6 vrf {vrf} interface', 'show ipv6 vrf all interface']
+                   'show ipv6 vrf {vrf} interface', 'show ipv6 vrf all interface',
+                   'show ipv6 interface']
```

Comment:
Please add logic with the `cli` function to handle this new command. See screenshot below of the `if` block that determines which command to send to the device.

![image](https://user-images.githubusercontent.com/33130026/132022050-3e665472-0593-40ce-a225-9a90b0037509.png)


You _might_ be able to simply add in an `elif not interface and not vrf:`, but if that doesn't work, then you'll have to make a small subclass for the 'show ipv6 interface' command

Reply:
Isn't this already covered by the else on line 1592? It will be used when there is no VRF or interface specified. 

```
            else:
                cmd = self.cli_command[2]
```

Reply:
Hmm. I'm not sure how to work with the if block for `show ipv6 vrf all interface` and `show ipv6 interface`  as neither will have a vrf or interface parameter specified. 

So currently `show ipv6 interface` would be hitting the else statement on line 1592/1593, but it would be running `show ipv6 vrf all interface`

Reply:
There was already a class for ShowIpv6Interface .. I added the cli command there instead.. think thats all I need to do ..?